_posts/2017-09-30-End-to-end-IoT.md

```diff
@@ -0,0 +1,188 @@
+
+This allowed us to demonstrate the capabilities of the LoRa network in a fun and engaging way.
+The application remained unchanged for the user, and was adapted to be more configurable:
+Setting a win chance (up to 100%) and different images/styling for different events.
+
+
+### The booze-o-meter
+The Booze-o-meter application is a drink disperser that relays the remaining level of fluid in the dispenser.
+It is a fun example to demonstrate how measuring a liquid of fill level of a container can be achieved.
+This idea can be applied to container in a whole range of different industries and use cases. 
+From oil tanks to garbage cans and containers.
+
+The application setup is extremely similar to the Slotmachine application. 
+The sensors relay their data via the MyThings platform to our backend, which in turns dispatches an event on a websocket so the frontend application can display the change.
+
+<p style="text-align: center;">
+  <img class="image fit" style="width: 350px; margin:0px auto;" alt="Booze-o-meter application" src="../img/end-to-end-iot/booze-1.jpg">
+   <img class="image fit" style="width: 350px; margin:0px auto;" alt="Booze-o-meter application" src="../img/end-to-end-iot/booze-2.jpg">
+</p>
+
+> The Booze-o-meter V1 test setup with regular water.
+
+
+#### V1
+The first version of the Booze-o-meter used three sensors that can detect a liquid through a thin plastic container. This allowed us to present the level in the container in a coarse way: 
+- FULL (initial state)
+- HIGH (sensor)
+- MEDIUM (sensor)
+- FULL (sensor)
+
+The sensors have a simple binary readout, true if liquid is detected, false if not.
+This data gets represented on the frontend application as the four states as mentioned above.
+
+#### V2
+<p style="text-align: center;">
+  <img class="image fit" style="width: 350px; margin:0px auto;" alt="Booze-o-meter application" src="../img/end-to-end-iot/booze-3.jpg">
+   <img class="image fit" style="width: 350px; margin:0px auto;" alt="Booze-o-meter application" src="../img/end-to-end-iot/booze-4.jpg">
+</p>
+
+> The Booze-o-meter V2 at Devoxx Belgium 2016 with actual liqour!
+
+The second version of the Booze-o-meter application allowed us to get a more detailed reading of the remaining fluid level in the container thanks to the addition of an ultrasonic sensor. 
+This sensor can measure the distance between itself and a surface, in this case the surface of the liquid in the container.
+
+The application was updated to support this more granular approach that is able to show the level in the container accurate to 1%.
+
+
+## Conclusion
+
+Our demo applications have served us well in bringing across the idea of LoRa to customers and other interested developers. 
+We will continue to evolve our demo applications to add new features and keep them up to date with newer and experimental techonolgies.
```

Comment:
You could add another small section at the end with all the useful links encountered in the blog post such as LoRa, MyThings and the GitHub repos.